doc/sphinx/language/core/primitive.rst

```diff
@@ -53,11 +53,8 @@ applications of these primitive operations.
 The extraction of these primitives can be customized similarly to the extraction
 of regular axioms (see :ref:`extraction`). Nonetheless, the :g:`ExtrOCamlInt63`
 module can be used when extracting to OCaml: it maps the Coq primitives to types
-and functions of a :g:`Uint63` module (including signed functions for
-:g:`Sint63` despite the name). That OCaml module is not produced by extraction.
-Instead, it has to be provided by the user (if they want to compile or execute
-the extracted code). For instance, an implementation of this module can be taken
-from the kernel of Coq.
+and functions of the :g:`Uint63` module from the `coq-primitive` library
+(including signed functions for :g:`Sint63` despite the name).
```

Comment:
I don't quite see why you say "despite the name".  What's the apparent inconsistency?

Reply:
The OCaml module for primitive ints is called `Uint63`, so it looks like it should contain only unsigned things.